### PR TITLE
feat: support auto build rpm package for dashboard

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

support auto build rpm package for dashboard

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by juzhiyuan** [comment]
please request reviews

**Review by gxthrj — APPROVED** [review]

**Review by tokers — APPROVED** [review]

File: .github/workflows/auto-build-rpm.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - name: Extract branch name
+      if: ${{ startsWith(github.ref, 'refs/heads/release/') }}
+      id: branch_env
+      shell: bash
+      run: |
+        echo "##[set-output name=version;]$(echo ${GITHUB_REF##*/})"
+
+    - name: Build rpm package
+      run: |
+        export VERSION=2.5
```
Comment (starsz) [inline .github/workflows/auto-build-rpm.yml:41]: Hard code ?

File: .github/workflows/auto-build-rpm.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        cd apisix-build-tools
+        make package type=rpm app=dashboard version=${VERSION} checkout=release/${VERSION}
+
+    - name: Run centos7 docker and mapping apisix into container
+      run: |
+        docker run -itd -v $PWD:/apisix-dashboard --name centos7Instance --net="host" docker.io/centos:7 /bin/bash
+
+    - name: Install rpm package
+      run: |
+        export VERSION=2.5
```
Comment (starsz) [inline .github/workflows/auto-build-rpm.yml:53]: Hard code ?

File: .github/workflows/auto-build-rpm.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        docker exec centos7Instance bash -c "cd apisix-dashboard && yum install -y ./apisix-build-tools/output/apisix-dashboard-${VERSION}-0.x86_64.rpm"
+        docker logs centos7Instance
+        # Dependencies are attached with rpm, so revert `make deps`
+        docker exec centos7Instance bash -c "cd /usr/local/apisix/dashboard/ && nohup ./manager-api &"
+
+    - name: Run test cases
+      run: |
+        ls -l
+        pwd
```
Comment (starsz) [inline .github/workflows/auto-build-rpm.yml:62]: Debug lines?

**Review by starsz — CHANGES_REQUESTED** [review]

**Review by starsz — APPROVED** [review]

**Pull request merged**